config/webpack.common.js

```diff
@@ -254,12 +254,11 @@ module.exports = function (options) {
         {
           test: /^(?!.*component).*\.css$/,
           use: [
-            'style-loader',
-            // isProd ? MiniCssExtractPlugin.loader : 'style-loader',
+            isProd ? MiniCssExtractPlugin.loader : 'style-loader',
```

Comment:
Did you confirm this works? I left it commented out as I had issues with it.

Reply:
Yes I tested running build:prod and then running a local server.